src/main/scala/karazin/scala/users/group/week2/homework/Homework.scala

```diff
@@ -35,39 +35,39 @@ object Homework:
 
     @targetName("addition")
     infix def +(that: Rational): Rational =
-      Rational(this.numer*that.denom+that.numer*this.denom,this.denom*that.denom)
+      Rational(this.numer * that.denom + that.numer * this.denom, this.denom * that.denom)
 
     @targetName("negation")
     infix def unary_- : Rational =
       -this
 
     @targetName("substraction")
     infix def -(that: Rational): Rational =
-      Rational(this.numer*that.denom-that.numer*this.denom,this.denom*that.denom)
+      Rational(this.numer * that.denom - that.numer * this.denom, this.denom * that.denom)
 
     @targetName("multiplication")
     infix def *(that: Rational): Rational =
-      Rational(this.numer*that.numer,this.denom*that.denom)
+      Rational(this.numer * that.numer, this.denom * that.denom)
 
     @targetName("devision")
     infix def /(that: Rational): Rational =
-      Rational(this.numer*that.denom,this.denom*that.numer)
+      Rational(this.numer * that.denom, this.denom * that.numer)
```

Comment:
What if `that.numer` is negative? Does it work?
Please review https://www.scala-lang.org/api/2.13.5/scala/math/index.html#signum(x:Int):Int